<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">border-radius: 5px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_div_content_1">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>110</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(41, 45, 56);
border-radius: 5px;
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_title_wid_1">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>35</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(39, 44, 54);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="labelBoxBlenderInstalation">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>TD Ameritrade Connection Information</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_content_wid_1">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QGridLayout" name="gridLayout_5">
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="1" column="0">
               <widget class="QLineEdit" name="lineEdit_apikey">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLineEdit {
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:hover {
	border: 2px solid rgb(64, 71, 88);
}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
                </property>
                <property name="text">
                 <string>C:\Users\hungd\Desktop\MACD\authentication\tdAmeritradeAPIKey</string>
                </property>
                <property name="placeholderText">
                 <string>C:\Users\MACD\constituents\constituents_example.csv</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="pushButton_apikey">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Segoe UI</family>
                  <pointsize>9</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton {
	border: 2px solid rgb(52, 59, 72);
	border-radius: 5px;	
	background-color: rgb(52, 59, 72);
}
QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}
QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}</string>
                </property>
                <property name="text">
                 <string>API Key</string>
                </property>
                <property name="icon">
                 <iconset resource="../uiResourceFiles/files.qrc">
                  <normaloff>:/16x16/icons/16x16/cil-folder-open.png</normaloff>:/16x16/icons/16x16/cil-folder-open.png</iconset>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>API Key File Path</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_6">
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="1" column="0">
               <widget class="QLineEdit" name="lineEdit_accountNumber">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLineEdit {
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:hover {
	border: 2px solid rgb(64, 71, 88);
}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
                </property>
                <property name="text">
                 <string>C:\Users\hungd\Desktop\MACD\authentication\tdAmeritradeAccountNumber</string>
                </property>
                <property name="placeholderText">
                 <string>C:\Users\MACD\constituents\constituents_example.csv</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="pushButton_accountNumber">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Segoe UI</family>
                  <pointsize>9</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton {
	border: 2px solid rgb(52, 59, 72);
	border-radius: 5px;	
	background-color: rgb(52, 59, 72);
}
QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}
QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}</string>
                </property>
                <property name="text">
                 <string>Account Number</string>
                </property>
                <property name="icon">
                 <iconset resource="../uiResourceFiles/files.qrc">
                  <normaloff>:/16x16/icons/16x16/cil-folder-open.png</normaloff>:/16x16/icons/16x16/cil-folder-open.png</iconset>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>Account Number File Path</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_8">
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="1" column="0">
               <widget class="QLineEdit" name="lineEdit_chromeDriver">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLineEdit {
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:hover {
	border: 2px solid rgb(64, 71, 88);
}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
                </property>
                <property name="text">
                 <string>C:\Users\hungd\Desktop\MACD\authentication\chromedriver</string>
                </property>
                <property name="placeholderText">
                 <string>C:\Users\MACD\constituents\constituents_example.csv</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="pushButton_chromeDriver">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Segoe UI</family>
                  <pointsize>9</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton {
	border: 2px solid rgb(52, 59, 72);
	border-radius: 5px;	
	background-color: rgb(52, 59, 72);
}
QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}
QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}</string>
                </property>
                <property name="text">
                 <string>Chrome Driver</string>
                </property>
                <property name="icon">
                 <iconset resource="../uiResourceFiles/files.qrc">
                  <normaloff>:/16x16/icons/16x16/cil-folder-open.png</normaloff>:/16x16/icons/16x16/cil-folder-open.png</iconset>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_4">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>Chrome Driver File Path</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_7">
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="1" column="0">
               <widget class="QLineEdit" name="lineEdit_token">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLineEdit {
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:hover {
	border: 2px solid rgb(64, 71, 88);
}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
                </property>
                <property name="text">
                 <string>C:\Users\hungd\Desktop\MACD\authentication\token</string>
                </property>
                <property name="placeholderText">
                 <string>C:\Users\MACD\constituents\constituents_example.csv</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="pushButton_token">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Segoe UI</family>
                  <pointsize>9</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton {
	border: 2px solid rgb(52, 59, 72);
	border-radius: 5px;	
	background-color: rgb(52, 59, 72);
}
QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}
QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}</string>
                </property>
                <property name="text">
                 <string>Token</string>
                </property>
                <property name="icon">
                 <iconset resource="../uiResourceFiles/files.qrc">
                  <normaloff>:/16x16/icons/16x16/cil-folder-open.png</normaloff>:/16x16/icons/16x16/cil-folder-open.png</iconset>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>Token File Path (Generated after first connection)</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../uiResourceFiles/files.qrc"/>
  <include location="../uiResourceFiles/files.qrc"/>
 </resources>
 <connections/>
</ui>
